tasks/test-tutorial

```diff
@@ -16,34 +16,46 @@ yarn install
 
 if [ -z "$1" ]
   then
-  echo "You have not supplied a path to a Redwood project."
+  echo "You have not supplied a path to a RedwoodJS project."
   echo "We will create one for you."
   echo "We will copy './packages/create-redwood-app/template' and the packages/*"
-
   TMP_DIR=$(mktemp -d -t redwood)
-
-  cd ../../packages/create-redwood-app
-  yarn babel-node src/create-redwood-app.js $TMP_DIR  --no-yarn-install
 else
   echo "You have supplied a path $1, we will not create a new "
   echo "Redwood project, we will use the app you have specified."
   TMP_DIR=$1
 fi
 
-if [ -z "$1" ]
+if [[ -z "$1"  || "$CREATE_RWJS_PROJECT" == "1" ]]
+  then
+  echo "Creating new RedwoodJS Project..."
+  cd ../../packages/create-redwood-app
+  yarn babel-node src/create-redwood-app.js $TMP_DIR  --no-yarn-install
+fi
+
+if [[ -z "$1"  || "$CREATE_RWJS_PROJECT" == "1" ]]
   then
   cd "$SCRIPTPATH/../"
-  # build all the packages
+  # build all the packages, but not the TypeScript
+  # since it is slow and we do not need it.
   yarn build:clean && yarn lerna run build:js
 fi
 
 cd $TMP_DIR
 
-if [ -z "$1" ]
+if [[ -z "$1" || "$CREATE_RWJS_PROJECT" == "1" ]];
```

Comment:
We still want to link when its not creating a new project right? Or was this intentional. No need to change this, just checking the logic for porting.

Reply:
This is a poor choice for a variable name.

Basically I want more flags to remix what happens here.

In e2e I want to:
- Create a new project in a specific folder.
-  I want to  build and link the framework files.

Locally I want to be able to:
- Create a new project in a specific folder.
- Do not build, link, and install the framework (leave it be.)
OR:
-  Create a new project in a random folder
-  I want to  build and link the framework files.

Reply:
Maybe we should just make these flags?

`./run-e2e-tests --new-project --rwt-link`

Reply:
Wanna make suggestions on my PR here? https://github.com/redwoodjs/redwood/pull/1896 so I can add them in?